test/formula-to-expression.test.js

```diff
@@ -108,6 +113,45 @@ describe('formulas', () => {
       ['upper', ['concat', 'here ', 'not there']]
     ]);
   });
+
+  test('3 % 2', () => {
+    const actual = formulaToExpression('3 % 2');
+    expect(actual).toEqual(['%', 3, 2]);
+  });
```

Comment:
Just noticed that in `expression-to-formula`, tests descriptions match the output, while here in `formula-to-expression` they match the input. I suppose we should choose one for consistency.

Reply:
I suppose it might be a good idea to explain what actually makes the test special, instead of copying the formula. I'll make a follow-up issue.